src/components/Header.js

```diff
@@ -0,0 +1,7 @@
+import styles from './Header.module.css';
+
+const Header = () => (
+  <h1 className={styles.header}>Todo List</h1>
+);
```

Comment:
Please I would like to know why you uses this type of styling `className={styles.header}`. i would prefer you use `className="item"`